config/karma/karma.conf.dev.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+var path = require("path");
+
+var MAIN_PATH = path.join(process.cwd(), "test/client/main.js");
+var POLYFILL_PATH = path.join(
+  path.dirname(require.resolve("core-js/package.json")), "es6/**/*.js"
+);
+var PREPROCESSORS = {};
+PREPROCESSORS[MAIN_PATH] = ["webpack"];
+PREPROCESSORS[POLYFILL_PATH] = ["webpack"];
+
+/*
+ * Karma Configuration: "dev" version.
+ *
+ * This configuration relies on a `webpack-dev-server` already running and
+ * bundling `webpack.config.test.js` on port 3001. If this is not running,
+ * then the alternate `karma.conf.js` file will _also_ run the webpack dev
+ * server during the test run.
+ */
+module.exports = function (config) {
+  config.set({
+    frameworks: ["mocha"],
+    reporters: ["spec"],
+    browsers: ["ChromeHeadlessCustom"],
+    customLaunchers: {
+      ChromeHeadlessCustom: {
+        base: 'ChromeHeadless',
+        // --no-sandbox for https://github.com/travis-ci/docs-travis-ci-com/pull/1671/files
+        flags: ['--no-sandbox']
+      }
+    },
+    basePath: ".", // repository root.
+    preprocessors: PREPROCESSORS,
+    files: [
+      // Sinon has issues with webpack. Do global include.
+      require.resolve("sinon/pkg/sinon"),
+
+      // Polyfills for PhantomJS in React 16.
```

Comment:
NIT: No longer needed for Phantom. _But_ if you test in IE9 you will need both. Can drop otherwise.